src/main/java/ch/hevs/cloudio/endpoint/CloudioEndpoint.java

```diff
@@ -465,6 +499,12 @@ class InternalEndpoint implements CloudioNodeContainer, MqttCallback, Runnable,
         private static final String PERSISTENCE_MQTT_UPDATE         = "cloudioPersistenceUpdate";
         private static final String PERSISTENCE_MQTT_LOG            = "cloudioPersistenceLog";
         private static final String PERSISTENCE_MQTT_LIFECYCLE      = "cloudioPersistenceLifecycle";
+        /*** TOKEN parameters******************************************************************************************/
+        private static final String AUTH_TOKEN = "ch.hevs.cloudio.endpoint.token";
+        private static final String AUTH_TOKEN_PROPERTIES_PATH = "ch.hevs.cloudio.endpoint.tokenPropertiesPath";
+        private static final String TOKEN_PROPERTIES_DEFAULT_PATH = "/etc/cloud.io/";
```

Comment:
`private static final String TOKEN_PROPERTIES_DEFAULT_PATH = "~/.config/cloud.io/";`

Default path should work even when not  running  as root.